.github/workflows/release-build-publish.yml

```diff
@@ -47,7 +47,7 @@ jobs:
 
       - name: Publish to Maven Central
         run: |
-          ./gradlew publish \
+          ./gradlew publishToSonatype closeAndReleaseSonatypeStagingRepository \
```

Comment:
@scirner22 I know we talked about not closing/releasing the staging repository for pre-releases and instead setting up our internal Nexus mirror to point to the staging repo, not sure how complex that setup is... I was thinking of just creating a prerelease to test this functionality and then we can determine how that mirror should be setup if we want to make the `closeAndReleaseSonatypeStagingRepository` portion of this command only conditionally applied

Reply:
I was able to set up the mirror so that the release only is 'closed' and not 'released' in the case of a prerelease, and I set up our nexus to have a proxy (added to the mirror group) pointing to https://s01.oss.sonatype.org/content/repositories/staging, which makes it available internally. I don't know if we will have a need to promote prereleases->releases at some point, but this seems to work for now